ui-admin/src/study/StudySettings.tsx

```diff
@@ -76,6 +76,16 @@ export function StudyEnvConfigView({ studyEnvContext, portalContext }:
       </label>
     </div>
 
+    <div>
+      <label className="form-label">
+        accepting proxy enrollment <InfoPopup content={
```

Comment:
can you start a zendesk page on proxy enrollment?  You'll probably just want to link that page, since the proxy handling will be up in the air for a while